airbyte-cdk/python/.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.78.6
+current_version = 0.78.7
```

Comment:
This is updated automatically by the bump version script that is part of the GitHub action. I'm not sure what happens when we try to manage this manually so I think it would be better to revert that line

Reply:
Thanks, good to know!